dev/how-to/maintain/tidb-binlog.md

```diff
@@ -6,41 +6,57 @@ category: reference
+    * Closing: You can only close a Pump process by using the `offline-pump` command in the binlogctl tool. After receiving the instruction, the Pump node sets its state to `closing` and stops receiving the binlog write request. The node continues providing binlog to Drainer nodes until all binlog data is consumed by Drainer nodes. Then, the Pump node sets its state to `offline` and exits the process.
+
+### Drainer
+
+* Starting: When started, the Drainer node tries to pull binlogs from all Pump nodes which are not in the `offline` state. If it fails to do so, it keeps trying.
+* Exiting: The Drainer node enters the `paused` or `offline` state before the process is exited normally; if the process is exited abnormally (caused by `kill -9`, process panic, crash), the Drainer node is still in the `online` state.
+    * Pausing: You can pause a Drainer process by using the `kill` command (not `kill -9`), entering `Ctrl` + `C` or using the `pause-drainer` command in the binlogctl tool. After receiving the instruction, the Drainer node sets its state to `pausing` and stops pulling binlogs from Pump nodes. After all threads are safely exited, the Drainer node sets its state to `paused` and exits the process.
+    * Closing: You can only close a Drainer process by using the `offline-drainer` command in the binlogctl tool. After receiving the instruction, the Drainer node sets its state to `closing` and stops pulling binlogs from Pump nodes. After all threads are safely exited, the Drainer node updates its state to `offline` and exits the process.
 
 For how to pause, close, check, and modify the state of Drainer, see the [binlogctl guide](#binlogctl-guide) as follows.
 
 ## `binlogctl` guide
 
 [`binlogctl`](https://github.com/pingcap/tidb-binlog/tree/master/binlogctl) is an operations tool for TiDB Binlog with the following features:
 
-* Obtaining the current `tso` of TiDB cluster
 * Checking the state of Pump or Drainer
-* Modifying the state of Pump or Drainer
 * Pausing or closing Pump or Drainer
+* Handling the abnormal state of Pump or Drainer
 
 ### Usage scenarios of `binlogctl`
 
-* It is the first time you run Drainer and you need to obtain the current `tso` of TiDB cluster.
-* When Pump or Drainer exits abnormally, its state is not updated and the service is affected. You can use this tool to modify the state.
 * An error occurs during data replication and you need to check the running status and state of Pump or Drainer.
 * While maintaining the cluster, you need to pause or close Pump or Drainer.
+* Pump or Drainer process is exited abnormally and the node state is not updated or is unexpected, which influences the business.
```

Comment:
For 业务 in our doc, use **application** instead of **business** generally. 

```suggestion
* Pump or Drainer process is exited abnormally, but the node state is not updated or is unexpected, which influences the application.
```